### PR TITLE
Update logrotate config file permissions

### PR DESCRIPTION
**Pull request by mserranom**
## Description

`logrotate` is failing to run `/etc/logrotate.d/sharelatex` due to existing `write` permisions.

```
error: Ignoring /etc/logrotate.d/sharelatex because of bad file mode - must be 0644 or 0444.
```

Updating the config permissions to be the same as other config fixes the issue.

Before:
 
```
$ ls -l /etc/logrotate.d
-rw-r--r-- 1 root root 120 Nov  2  2017 alternatives
-rw-r--r-- 1 root root 173 Apr 20  2018 apt
-rw-r--r-- 1 root root 112 Nov  2  2017 dpkg
-rw-r--r-- 1 root root 329 Apr  6  2018 nginx
-rw-rw-r-- 1 root root 123 Jan 12 13:06 sharelatex
-rw-r--r-- 1 root root 609 Aug 16  2018 syslog-ng
```
After:
```
$ ls -l /etc/logrotate.d
-rw-r--r-- 1 root root 120 Nov  2  2017 alternatives
-rw-r--r-- 1 root root 173 Apr 20  2018 apt
-rw-r--r-- 1 root root 112 Nov  2  2017 dpkg
-rw-r--r-- 1 root root 329 Apr  6  2018 nginx
-rw-r--r-- 1 root root 123 Jan 12 13:06 sharelatex
-rw-r--r-- 1 root root 609 Aug 16  2018 syslog-ng
```

### CODE REVIEW

**Review by das7pad — APPROVED** [review]
Interesting find. Here are some data points:

TL;DR: some time between release 2.3.0 (April 2020) and 2.4.0 (June 2020) the file permissions changed



```
overleaf$ rm logrotate/sharelatex 
overleaf$ git reset --hard
HEAD is now at 8647127 Merge pull request #826 from overleaf/jpa-fix-contacts-logging
overleaf$ ls -l logrotate/sharelatex 
-rw-r--r-- 1 me me 121 Jan 13 16:31 logrotate/sharelatex
```

---

```
$ docker run --rm --entrypoint ls sharelatex/sharelatex:v1.2.1 -l /etc/logrotate.d/sharelatex
-rw-r--r-- 1 root root 121 Mar  8  2018 /etc/logrotate.d/sharelatex
```

``` 
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.0.0 -l /etc/logrotate.d/sharelatex
-rw-r--r-- 1 root root 121 Oct  8  2019 /etc/logrotate.d/sharelatex
```

```
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.1.0 -l /etc/logrotate.d/sharelatex
-rw-r--r-- 1 root root 121 Nov 21  2019 /etc/logrotate.d/sharelatex
``` 

``` 
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.2.0 -l /etc/logrotate.d/sharelatex
-rw-r--r-- 1 root root 121 Nov 21  2019 /etc/logrotate.d/sharelatex
``` 

```
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.3.0 -l /etc/logrotate.d/sharelatex
-rw-r--r-- 1 root root 121 Apr  7  2020 /etc/logrotate.d/sharelatex
```

```
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.4.0 -l /etc/logrotate.d/sharelatex
-rw-rw-r-- 1 root root 121 Jul  8  2020 /etc/logrotate.d/sharelatex
```

```
$ docker run --rm --entrypoint ls sharelatex/sharelatex:2.5.0 -l /etc/logrotate.d/sharelatex
-rw-rw-r-- 1 root root 121 Oct 26 10:08 /etc/logrotate.d/sharelatex
```

---

```
$ git clone https://github.com/overleaf/overleaf
$ ls -l overleaf/logrotate/sharelatex 
-rw-rw-r-- 1 me me 121 Jan 13 16:32 overleaf/logrotate/sharelatex
``` 
```
$ curl https://github.com/overleaf/overleaf/archive/master.tar.gz -Ls | tar -xz --strip-components=1
$ ls -l logrotate/sharelatex 
-rw-rw-r-- 1 me me 121 Jan 12 11:54 logrotate/sharelatex
```
